sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobAsyncClientBase.java

```diff
@@ -115,6 +117,25 @@ protected BlobAsyncClientBase(HttpPipeline pipeline, String url, BlobServiceVers
         this.blobName = blobName;
         this.snapshot = snapshot;
         this.customerProvidedKey = customerProvidedKey;
+        this.blobPropertyHelper = null;
```

Comment:
I don't see this `blobPropertyHelper` being set anywhere else in this class and it's private. So, not sure how this will be used.